server/server.go

```diff
@@ -94,6 +98,26 @@ func New(c *Config, log internal.Logger) *Server {
 	s.JoinMessage(c.Server.JoinMessage)
 	s.QuitMessage(c.Server.QuitMessage)
 
+	if _, err := os.Stat(c.Resources.Folder); os.IsNotExist(err) {
+		_ = os.Mkdir(c.Resources.Folder, 0777)
+	}
+	resources, err := os.ReadDir(c.Resources.Folder)
+	if err != nil {
+		panic(err)
+	}
+	for _, entry := range resources {
+		if entry.IsDir() {
```

Comment:
Why ignore directories here? Gophertunnel's resource pack implementation can parse directories too.

Reply:
Thanks for the review! Will fix!